linode/manager
# fix(PrimaryNav): M3-6162 - Improve new branded logo animation and transition

## Description 📝
- The transition for the new Akamai logo used for the brand update was a little janky when opening and closing the nav bar. This adds a smoother animation in addition to a slight fade-in of the logo when opening.
- This also fixes an issue where the nav bar would transition off-screen when using a keyboard to tab.
- Added some accessibility improvements: Nav bar links will now convey which if they are currently active

## Preview 📷
| Before      | After |
| ----------- | ----------- |
| <video src="https://user-images.githubusercontent.com/125309814/219542741-1e4444fb-00d4-45b0-bf4b-200869749a6e.mp4" /> | <video src="https://user-images.githubusercontent.com/125309814/220502069-781d4117-805c-4b66-8255-59f3fcee047a.mp4" /> |

| Before      | After |
| ----------- | ----------- |
| <video src="https://user-images.githubusercontent.com/125309814/219543013-b5b40706-b33b-4fd3-b94c-6108f4d0523b.mp4" /> | <video src="https://user-images.githubusercontent.com/125309814/220502229-cd8788fd-1281-43f0-882f-21932d451216.mp4" /> |

## Accessibility Improvements
![M3-6162-solution-3](https://user-images.githubusercontent.com/125309814/219548343-1a901087-d419-4a27-86ed-5b60616f347d.gif)

## How to test 🧪
1. Checkout branch
2. Go to http://localhost:3000/linodes
3. Ensure "Brand Update" feature flag is enabled using DevTools
4. Observe that logo transition should be much smoother
